programs/server/config.xml

```diff
@@ -562,7 +563,9 @@
     <!-- Work factor for bcrypt_password authentication type-->
     <bcrypt_workfactor>12</bcrypt_workfactor>
 
-    <!-- Complexity requirements for user passwords. -->
+    <!-- Complexity requirements for user passwords.
+         Note: ClickHouse Cloud https://clickhouse.cloud/ is always configured for strong passwords.
```

Comment:
@alexey-milovidov
For unauthorized person, the https://clickhouse.cloud/ link leads to SQL Console Login page.
Probably it is better to direct people to a landing page: https://clickhouse.com/cloud

Reply:
https://github.com/ClickHouse/ClickHouse/pull/75266